docs/contributing/code-style.md

```diff
@@ -180,33 +173,14 @@ in both development and production.
 
 ## JS array/object manipulation
 
-For generic functions that operate on arrays or JavaScript objects, you
-should generally use [Underscore](http://underscorejs.org/). We used to
-use jQuery's utility functions, but the Underscore equivalents are more
-consistent, better-behaved and offer more choices.
-
-A quick conversion table:
-
-       $.each → _.each (parameters to the callback reversed)
-       $.inArray → _.indexOf (parameters reversed)
-       $.grep → _.filter
-       $.map → _.map
-       $.extend → _.extend
-
-There's a subtle difference in the case of `_.extend`; it will replace
-attributes with undefined, whereas jQuery won't:
-
-       $.extend({foo: 2}, {foo: undefined});  // yields {foo: 2}, BUT...
-       _.extend({foo: 2}, {foo: undefined});  // yields {foo: undefined}!
-
-Also, `_.each` does not let you break out of the iteration early by
-returning false, the way jQuery's version does. If you're doing this,
-you probably want `_.find`, `_.every`, or `_.any`, rather than 'each'.
-
-Some Underscore functions have multiple names. You should always use the
-canonical name (given in large print in the Underscore documentation),
-with the exception of `_.any`, which we prefer over the less clear
-'some'.
+For functions that operate on arrays or JavaScript objects, you should
+generally use modern ECMAScript primitives such as `for … of` loops,
+`Array.prototype.{entries, every, filter, find, indexOf, map, some}`,
+`Object.{assign, entries, keys, values}`, spread syntax, and so
+on. Our Babel configuration automatically transpiles and polyfills
+these using `core-js` when necessary. We used to use the
```

Comment:
One of the things I'll miss about underscore is being able to quickly scan the left panel in its docs (https://underscorejs.org/) and pretty quickly find the function I need.  It would be nice to provide a similar link for `Array`.  I have to admit that I don't know what the most reliable reference for JS methods is, and I usually end up at something like w3schools via google, which I assume is mostly trustworthy.

Reply:
I usually turn to MDN. Added lots of links.